.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: "CI"
+on: # yamllint disable
+  push:
+    branches:
+      - "main"
+  pull_request:
+  release:
+    types: [published]
+
+jobs:
+  lint:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "CHECKOUT REPOSITORY CODE"
+        uses: "actions/checkout@v2"
+      - name: "SETUP ENVIRONMENT"
+        uses: "networktocode/gh-action-setup-poetry-environment@v4"
+      - name: "RUN YAMLLINT ON ANSIBLE PLAYBOOK"
+        run: "poetry run yamllint pb_nautobot_install.yml"
+      - name: "RUN HADOLINT ON DOCKERFILE"
+        uses: "hadolint/hadolint-action@v3.1.0"
+        with:
+          dockerfile: Dockerfile
```

Comment:
Any reason to not lint the whole repo?

Reply:
Hadolint lints Dockerfiles for best practices. :)

Reply:
:doh: Missed we do the same in Core. What about yamllint though?